pty_windows.go

```diff
@@ -28,11 +29,11 @@ type WindowsTty struct {
 }
 
 var (
-	// NOTE(security): as noted by the comment of syscall.NewLazyDLL and syscall.LoadDLL
-	// 	user need to call internal/syscall/windows/sysdll.Add("kernel32.dll") to make sure
-	//  the kernel32.dll is loaded from windows system path
+	// NOTE(security): As noted by the comment of syscall.NewLazyDLL and syscall.LoadDLL
+	//                 user need to call internal/syscall/windows/sysdll.Add("kernel32.dll") to make sure
+	//                 the kernel32.dll is loaded from windows system path.
 	//
-	// ref: https://pkg.go.dev/syscall@go1.13?GOOS=windows#LoadDLL
+	// Ref: https://pkg.go.dev/syscall@go1.13?GOOS=windows#LoadDLL
 	kernel32DLL = windows.NewLazyDLL("kernel32.dll")
```

Comment:
Isn't it a concern that this will be executed by anyone importing the package, even if they don't use it?

Reply:
I don't think it's a concern since the windows package also does it this way. We are not actually loading the DLL. The DLL is loaded when we call the Find method if it is not already loaded.